enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -58,7 +63,7 @@ func (i *InvocationStatService) getAggColumn(reqCtx *ctxpb.RequestContext, aggTy
 
 func (i *InvocationStatService) GetTrendBasicQuery(timezoneOffsetMinutes int32) string {
 	q := ""
-	if i.olapdbh == nil {
+	if i.olapdbh == nil || !*readFromOLAPDBEnabled {
```

Comment:
can you pull this out to a function

func (i *ISS) OLAPEnabled() bool {
  return i.olapdbh != nil && *readFromOLAPDBEnabled
}

and then use this everywhere you have the || now?

Reply:
Done